### PR TITLE
Revert "Revert "Added fairdatapoint harvester""

### PR DESCRIPTION
**Pull request by hcvdwerf**
This reverts commit 02537041ee2986a0b4494205bee2ecae7181399a.

### CODE REVIEW

File: ckan/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,12 @@ RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userpo
 RUN  pip3 install -e git+https://github.com/DataShades/ckanext-oidc-pkce.git@v0.3.1#egg=ckanext-oidc-pkce && \
      pip3 install -r ${APP_DIR}/src/ckanext-oidc-pkce/requirements.txt
 
+RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-fairdatapoint.git@main#egg=ckanext-fairdatapoint && \
+     pip3 install -r ${APP_DIR}/src/ckanext-fairdatapoint/requirements.txt
+
+RUN  pip3 install -e git+https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-civity.git@main#egg=ckanext-civity
```
Comment (brunopacheco1) [inline ckan/Dockerfile:24]: If only `fairdatapointharvester` is needed, can you please remove ckanext-civity from the dependencies?

**Review by brunopacheco1 — CHANGES_REQUESTED** [review]
Hey @ha

File: ckan/Dockerfile (hunk 1 above)
Comment (hcvdwerf) [reply ckan/Dockerfile]: Yes done. see comments slack. Refactor the code for now to make fairdatapoint not depended on it

**Comment by hcvdwerf** [comment]
> Hey @ha

He Bru

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**